articles/governance/blueprints/create-blueprint-rest-api.md

```diff
@@ -4,7 +4,7 @@ description: In this quickstart, you use Azure Blueprints to create, define, and
 ms.date: 08/17/2021
 ms.topic: quickstart
 ---
-# Quickstart: Define and Assign an Azure Blueprint with REST API
+# Quickstart: Define and Assign an Azure Blueprints with REST API
```

Comment:
```suggestion
# Quickstart: Define and assign an Azure blueprint with REST API
```

Reply:
Here, the word "an" indicates that "blueprint" is a noun, an instance of the service, so it's lowercase and singular.